Dockerfiles/cluster-agent/README.md

```diff
@@ -136,16 +120,46 @@ Create a [service](../manifests/cluster-agent/datadog-cluster-agent_service.yaml
 Ensure an auth_token is properly shared between the agents.
 Confirm the [RBAC rules](../manifests/rbac) are properly set.
 
-In the Node Agent, set the env var `DD_CLUSTER_AGENT` to true.
+In the Node Agent, set the env var `DD_CLUSTER_AGENT_ENABLED` to true.
 
 The env var `DD_KUBERNETES_METADATA_TAG_UPDATE_FREQ` can be set to specify how often the Node Agents hit the Datadog Cluster Agent.
 You can disable the kubernetes metadata tag collection with `DD_KUBERNETES_COLLECT_METADATA_TAGS`.
 
-#### HPA
+#### Custom Metrics Server
+
+The Datadog Cluster Agent implements the External Metrics Provider's interface in alpha.
```

Comment:
`The Datadog Cluster Agent implements the External Metrics Provider interface and is currently in alpha.`